test/celeritas/field/Steppers.test.cc

```diff
@@ -75,14 +75,14 @@ class SteppersTest : public Test
         auto stepper = make_mag_field_stepper<StepperT>(
             field, units::ElementaryCharge{-1});
         // Test parameters and the sub-step size
-        real_type hstep = 2.0 * constants::pi * param.radius / param.nsteps;
+        real_type hstep = 2 * constants::pi * param.radius / param.nsteps;
 
         for (unsigned int i : range(param.nstates))
         {
             // Initial state and the epected state after revolutions
             OdeState y;
-            y.pos = {param.radius, 0.0, i * 1.0e-6};
-            y.mom = {0.0, param.momentum_y, param.momentum_z};
+            y.pos = {param.radius, 0, i * real_type{1e-6}};
```

Comment:
What's `1e-6` here?

Reply:
A small bump? You'll have to ask @whokion .

Reply:
Well the related question is whether or not it needs to be adjusted for `double` vs `float`.

Reply:
The test passed without it so I guess not?

Reply:
> A small bump? You'll have to ask @whokion .

This extra addition is just not to make the operation identical in the loop (i.e, may avoid that the compile is smart enough and execute the loop operation just once as the input/result is same - too smart?).  Anyway, it was introduced when we actually evaluated performance for different stepper cases, which is no longer the case now.